proto/v1/review_service.proto

```diff
@@ -177,6 +177,9 @@ message ApprovalTemplate {
   ApprovalFlow flow = 1;
   string title = 2;
   string description = 3;
+  // The name of the creator.
+  // Format: users/{user}. {user} is a system-generated unique ID.
+  string creator_name = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
nit: just call it creator for simplicity? We can keep the consistency in the future.

Reply:
LGTM, updated